includes/rest-api/class-wp-job-manager-data-stores-status.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Declaration of our Status Data Store
+ *
+ * @package WPJM/REST
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class WP_Job_Manager_Data_Stores_Status
+ */
+class WP_Job_Manager_Data_Stores_Status extends WPJM_REST_Data_Store_Abstract
+	implements WPJM_REST_Interfaces_Data_Store {
+
+	/**
+	 * Get all the models (taking into account any filtering)
+	 *
+	 * @param  WPJM_REST_Interfaces_Model|null $filter A filter.
+	 * @return WPJM_REST_Model_Collection
+	 */
+	public function get_entities( $filter = null ) {
+		return new WPJM_REST_Model_Collection( array( $this->get_entity() ) );
+	}
+
+	/**
+	 * Get a Model Using it's unique identifier
+	 *
+	 * @param  mixed $id The id of the entity.
+	 * @return WPJM_REST_Interfaces_Model
+	 */
+	public function get_entity( $id = null ) {
```

Comment:
When I ran the tests in PHP 5.2, I got an error that this signature does not match `WPJM_REST_Interfaces_Data_Store`. Looks like pre~5.3.9 was more strict about interface signature matching.